examples/snapshot_json_string_example

```diff
@@ -0,0 +1 @@
+{"application":"application-sample","components":[{"name":"component-sample","containerImage":"quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9","source":{"git":{"url":"[git@github.com](mailto:git@github.com):redhat-appstudio/integration-service.git\n","revision":"sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c"}}
```

Comment:
Do we need this string version file or can we just use the .json file to show the example?

Reply:
The parameter [SNAPSHOT](https://github.com/redhat-appstudio/integration-service/blob/main/tekton/integration_pipeline.go#L152C2-L152C2) is to contain the Snapshot json string.

Reply:
Ok, let's keep it that way then.